contracts/LATokenMinter.sol

```diff
@@ -133,7 +133,7 @@ contract LATokenMinter {
         token = LATToken(_LATTokenAddress);
 
         numberOfDays = 5 * 365; // 5 years
-        startTime = 1503399600; // 22 august 2017 11:00 GMT+0;
+        startTime = 1661166000; // 22 august 2017 11:00 GMT+0;
```

Comment:
comment is wrong 😭

Reply:
Added an issue here: https://github.com/BlockchainLabsNZ/LAToken-Contracts-Audit/issues/31